pandas/io/formats/style.py

```diff
@@ -1873,14 +1858,18 @@ def hide_index(
             self.hidden_rows = hrows  # type: ignore[assignment]
         return self
 
-    def hide_columns(self, subset: Subset | None = None) -> Styler:
+    def hide_columns(
+        self,
+        subset: Subset | None = None,
+        levels: Level | list[Level] | None = None,
```

Comment:
umm didn't we change this to `level` ?

Reply:
have changed this, and from the last pr (`hide_index`) also.